staging/src/k8s.io/code-generator/cmd/validation-gen/validators/openapi.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2024 The Kubernetes Authors.
```

Comment:
This file was broken up in dev branch - do you want to do that here or later?

Reply:
Resolving this in https://github.com/kubernetes/kubernetes/pull/130730